org_fedora_oscap/common.py

```diff
@@ -66,13 +66,20 @@ def N_(string): return string
 TARGET_CONTENT_DIR = "/root/openscap_data/"
 
 SSG_DIR = "/usr/share/xml/scap/ssg/content/"
-SSG_CONTENT = "ssg-rhel7-ds.xml"
-if constants.shortProductName != 'anaconda':
-    if constants.shortProductName == 'fedora':
-        SSG_CONTENT  = "ssg-fedora-ds.xml"
-    else:
-        SSG_CONTENT = "ssg-%s%s-ds.xml" % (constants.shortProductName,
-                                            constants.productVersion.strip(".")[0])
+
+# Enable patches that set the content name at package-time
```

Comment:
What is  "low-risk monkey-patching"? How can I use it? How do I set the content name at package-time?

Reply:
My thought was that the packaging step could invoke `sed` to set e.g. `DEFAULT_SSG_CONTENT_NAME = 'ssg-rhv4-ds.xml'`, in other words, we one could set the content filename manually especially in cases when we know autodetection won't work.